registration-app/package.json

```diff
@@ -2,25 +2,30 @@
   "name": "registration-app",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": "^22.0.0"
+  },
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
+    "next": "15.0.2",
     "react": "19.0.0-rc-02c0e824-20241028",
-    "react-dom": "19.0.0-rc-02c0e824-20241028",
-    "next": "15.0.2"
+    "react-dom": "19.0.0-rc-02c0e824-20241028"
```

Comment:
Afaik React v19 is considered an unstable version. Rena mentioned something about this. Should we continue with it, or would it be better to wait for a stable release? imo we keep running with v19.

Reply:
rc stands for release candidate - usually comes between the last beta and before v19.0.0, but I'm not familiar with react release policy and process.

Either way, this PR doesn't change the react version, it only adds a few dependencies for ESLint to work properly. npm sorted the dependencies when I used the command to add them, hence the mess here.